src/main/java/frc/robot/command/undertaker/EjectNote.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.command.undertaker;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.Undertaker;
+
+public class EjectNote extends Command {
+    
+     private Undertaker undertaker;
+
+
+  /** Creates a new IntakeNote. */
```

Comment:
CPFTM - should be `EjectNote`

Reply:
You could also remove this comment and the matching on in `IntakeNote`, i think they are unnecessary